### DISCUSSION TITLE
Licenses

### DISCUSSION THREAD

**Discussion by EstherPlomp** [EstherPlomp/TNW-RDM-101#9] [General]
Based on the information you read in [Data and Code Licences](https://estherplomp.github.io/TNW-RDM-101/07-FAIR-principles.html#data-and-code-licences), please post your license preference for data and/or code below.

**Comment by RamonvanValderen** [comment]
For my research I would prefer to have the CC BY-NC-ND 4.0 license. This license requires that reusers give credit to the creator. It allows reusers to copy and distribute the material in any medium or format in unadapted form and for noncommercial purposes only. The main reason for this is the collaboration with have with a non-profit organization.

**Reply by EstherPlomp** [reply]
Hi @RamonvanValderen! Just a comment from my side that ND (Non-Derivative) is quite restrictive. I tend to recommend this license for interview data where the context is really important. If people can't reuse parts of the dataset it might hamper reuse, and also ND does not allow any translation of the work/data. 

That said, it is of course, up to you what license you would like to use!

**Reply by EstherPlomp** [reply]
And an added note: 
You also indicated that you work with code - will you release any scripts and what license will you choose or this? The CC licenses (unless it is the CC0) are generally not applicable for code because it is executable and a bit more complex to reuse compared to data.

**Comment by bertvanherck** [comment]
I would use the CC BY-NC 4.0 license, since this license requires that reusers give credit to the creator, while it allows reusers to distribute, remix, adapt, and build upon the material in any medium or format (for noncommercial purposes only). In this way, our work gets the credit it deserves, but it allows others to use it and implement changes in a flexible way.

**Reply by EstherPlomp** [reply]
Thanks for sharing your license choice for data! You also indicated that you work with code - will you release any scripts and what license will you choose or this? The CC licenses (unless it is the CC0) are generally not applicable for code because it is executable and a bit more complex to reuse compared to data.

**Comment by bejuarez96** [comment]
Since my research in funded by the government and it is in collaboration with a non-profit organization, for the results of my research I would use the CC BY-NC-SA 4.0 license. This license requires that reusers give credit to the creator, allows reusers to distribute, remix, adapt, and build upon the material in any medium or format, for noncommercial purposes only. But if others modify or adapt the material, they must license the modified material under identical terms.

**Comment by SDev5** [comment]
I would use the CC BY-NC 4.0 license for data, presentations and articles, which allows others to remix, adapt and build upon the work in a non-commercial way, while crediting the original creators. I am inclined to say that the non-commerciality is an important aspect for me, because we do science for non-commercial purposes, but at the same time I am not sure if this restriction is really necessary.  Scientific knowledge should be public and if someone can use that to create a commercially viable technology, then that is a good thing I would say... So I'm not sure about NC.

**Reply by EstherPlomp** [reply]
If you're not sure about things it is easier to first add more restrictions (so keep the NC part), and then later adjust to remove that part if you change your mind. Putting in more restrictions at a later time is more complicated if you already release materials under a less restrictive license. 

It is indeed not that straight forward - What if a PhD at a different university is partly funded via a company? This already falls under commercial re-use. They can always approach you directly to ask for permission to reuse, so it doesn't mean that commercial reuse is never possible - it just poses another barrier. I hope that makes sense and provides some more food for thought!

Additionally: You also indicated that you work with code - will you release any scripts and what license will you choose or this? The CC licenses (unless it is the CC0) are generally not applicable for code because it is executable and a bit more complex to reuse compared to data.

**Reply by SDev5** [reply]
Thank you for the advice! 

Regarding code, I might supply analysis or simulation scripts with the data publication of a paper, but currently have no plans of releasing more serious software packages. In any case, I would follow the standard in our group, which is the MIT license.

**Comment by MBeatrizDiniz** [comment]
I would use CC BY-SA 4.0. 
This license required reusers to give credit to the creator and it also allows reusers to distribute, adapt, and remix the material. My work has the potential for industrial applications, so it does make sense to be open for commercial purposes.

**Reply by EstherPlomp** [reply]
Makes sense indeed! It is also especially nice to keep the SA part in there if you work with commercial partners. That ensures that they will have to release their work under the same open license if they reuse and redistribute the original work :). You also indicated that you work with code - will you release any scripts and what license will you choose or this? The CC licenses (unless it is the CC0) are generally not applicable for code because it is executable and a bit more complex to reuse compared to data.

**Comment by maverheijen** [comment]
I would use CC BY-NC-SA 4.0 for my research. This requires users to credit the creator and users are then allowed to distribute, remix, adapt, and build upon the material in any medium or format, for noncommercial purposes only. I would prefer noncommercial purposes, because I don't think industry should just be able to make a profit of of other people's work. Because I do strongly believe in the benefits of other people building upon your work, I would want users to share their work under similar terms, so that it does stay non-commercial.

**Reply by EstherPlomp** [reply]
CC BY-NC-SA 4.0 would indeed be the best choice for your preferences! Will you also share code/software and what license will you choose for this?

**Comment by begum94** [comment]
I will use CC-BY 4.0 license which requires attribution/credit for the original creation, while at the same time ensures broadest possible re-use. I chose this license based on DMP plan of my phd project.

**Comment by MiriamCammaert** [comment]
Personally, I would use the CC BY-NC-SA 4.0 license. This gives the creators credit for the work while allowing reusers to remix, adapt, distribute, and build upon the data in any format. For now, I would like to keep the data for noncommercial purposes only considering the NWO grant for my PhD and the goal of the project. However, this NC license might be debatable, so that is definitely something to give further consideration in the future.

**Reply by EstherPlomp** [reply]
Commercial reuse can indeed be complicated! See also the thread by @SDev5 above for some thoughts. 

You also indicated that you will be using code/software to analyse data: will you share these scripts and if so - what is your preferred license for this? We have different licenses for software as it differs from data in complexity and reuse. See the [resources in the course materials](https://estherplomp.github.io/TNW-RDM-101/07-FAIR-principles.html#data-and-code-licences) for more information.

**Comment by TamaraJankovic03** [comment]
Personally, I would use the CC BY-SA license for my work. I think that this is the fairest license for my project because it allows reusers to remix, adapt and build upon the work even for commercial purposes, as long as they credit the creator and license their new work under identical terms. However, the possible licenses will be the topic for discussion at the next progress meeting with my supervisors.

**Reply by EstherPlomp** [reply]
Indeed - licenses should be chosen in collaboration with any collaborators and especially your supervisors!

**Comment by MarijnVDBr** [comment]
I think I would use CC BY-SA 4.0 for my work. It allows others to remix, adapt, or build upon your work and give credits to the creator. However, I am not completely sure whether to make it open for commercial purposes or not. I would like to increase the accessibility of my data as much as possible, but it will have to be determined together with the funders of my project (NWO).

**Reply by EstherPlomp** [reply]
NWO stimulates collaboration with industry, so I don't think they will have anything against commercial reuse. After all, industry is also part of the public which is providing funding for research. I can relate to the hesitations though! See also the threads by @MiriamCammaert and @SDev5 above for some thoughts about commercial re-use. 

You also indicated that you work with code/software in your research: Will you be able to share any of this and what would be your preferred license?

**Comment by rooswouters** [comment]
I think I would use the CC BY-NC-SA license. This because, while I would like other to be able to remix, adapt or build upon my work, for now my work is also still considered potentially commercially exploitable. Therefore, I like the additional restriction of not allowing commercial applications.  However, this is something I would have to further discuss with my supervisors and collaborators.

**Reply by EstherPlomp** [reply]
Licenses are definitely something to discuss with your collaborators and supervisors! When you head into this discussion you can also argue that even if you apply a NC license to the work - people can always approach the research team to ask for permission to reuse. If you give them written permission to use the work it works similarly to adding a license to a public dataset - the two are not mutually exclusive because you remain the rights holder.

I hope that makes sense - if not please let me know!

See also the thread by @SDev5 for some more thoughts on commercial reuse.

**Comment by sweerdenburg1** [comment]
I personally would like to use the CC BY 4.0 license, since I would like my work to be used, shared and extended upon as long as I receive credit for creating the data. I do need to check whether my consortium agrees with this license or that we need a more restrictive license before publication.

**Reply by EstherPlomp** [reply]
Definitely check license preferences with your collaborators! 

Additionally - you indicated that you also use code in your research: will you be able to share any of the code/scripts/software and if so - what is your preferred license? The licenses for software are a bit more complicated because software is more complex to reuse. See the [resources in the course materials](https://estherplomp.github.io/TNW-RDM-101/07-FAIR-principles.html#data-and-code-licences) for more information on software licenses.

**Comment by annickteepe** [comment]
I have never thought about these licenses and before today didn't know they actually existed on research papers! I think I would like to use the CC BY-SA 4.0 for my work. It allows others to build upon my work, but it does give the credits to the creator.

**Reply by EstherPlomp** [reply]
Yes indeed - research papers should be licensed to clearly indicate what reuse is possible and the vast majority of publishers assign licenses :) 

Additionally - you indicated that you also use code in your research: will you be able to share any of the code/scripts/software and if so - what is your preferred license? The licenses for software are a bit more complicated because software is more complex to reuse. See the [resources in the course materials](https://estherplomp.github.io/TNW-RDM-101/07-FAIR-principles.html#data-and-code-licences) for more information on software licenses.

**Comment by Azinkarimzad** [comment]
I would prefer CC BY-NC-SA for my work. This license allows others to remix, adapt, and build upon my work non-commercially, as long as they credit me and license their new creations under identical terms. I have a horror story to share where my former academic position founder used my newly developed method for commercial gain and filed a patent without my consent. Therefore, I believe this license is more suitable for my work.

**Reply by EstherPlomp** [reply]
Ai... sorry to hear about this! 

Note that once you share work publicly it will also be impossible for others to use it for patent applications. To apply for a patent all of the supporting materials must remain under restricted access. 

Additionally - you indicated that you also use code in your research: will you be able to share any of the code/scripts/software and if so - what is your preferred license? The licenses for software are a bit more complicated because software is more complex to reuse. See the [resources in the course materials](https://estherplomp.github.io/TNW-RDM-101/07-FAIR-principles.html#data-and-code-licences) for more information on software licenses.

**Comment by rwmulder** [comment]
Based on the License chooser, I think I would need the CC BY-SA 4.0 license. This way my work can be freely build upon by others while still giving me credit, even when the work is adapted. I am not sure if this is in line with the requirements of my NWO grant.

**Reply by EstherPlomp** [reply]
This is in line with NWO requirements! They suggest to use CC-BY but they do not place any formal requirements on your license choice right now, so you are free to choose this. In the future this might change if they change their policies, but that should not be applicable to existing grants!

Additionally - you indicated that you also use code in your research: will you be able to share any of the code/scripts/software and if so - what is your preferred license? The licenses for software are a bit more complicated because software is more complex to reuse. See the [resources in the course materials](https://estherplomp.github.io/TNW-RDM-101/07-FAIR-principles.html#data-and-code-licences) for more information on software licenses.

**Comment by phvree** [comment]
From the video and the license chooser I would think the best license for my work is CC BY-SA 4.0, since I would everybody to use and build upon my data, but would like to receive credit. So far, I do not think it is a big problem if anybody uses my data to do something commercially, so long as I get credit, although this could change maybe depending on my research findings :). Not sure if this license is normal in my field

**Reply by EstherPlomp** [reply]
You can also choose different licenses for different datasets! The only thing that is more difficult is to choose a more restrictive license if you already released materials before under a less restrictive license. The other way around is not a problem. (I hope this makes sense!)

Have you tried searching for available datasets from your field via google or the [4TU.ResearchData](https://data.4tu.nl/info//en/) or [zenodo](https://zenodo.org/) repository? You can check which licenses they are using. CC BY-SA is a commonly used license in any case!

Additionally - you indicated that you also use code in your research: will you be able to share any of the code/scripts/software and if so - what is your preferred license? The licenses for software are a bit more complicated because software is more complex to reuse. See the [resources in the course materials](https://estherplomp.github.io/TNW-RDM-101/07-FAIR-principles.html#data-and-code-licences) for more information on software licenses.

**Comment by MaximDeSmet** [comment]
According to the license chooser, I would use the CC BY 4.0 license. Given that proper credit is in place, others may use, mix and build upon my results and exploit it commercially.

**Reply by EstherPlomp** [reply]
Sounds good! 

Additionally - you indicated that you also use code in your research: will you be able to share any of the code/scripts/software and if so - what is your preferred license? The licenses for software are a bit more complicated because software is more complex to reuse. See the [resources in the course materials](https://estherplomp.github.io/TNW-RDM-101/07-FAIR-principles.html#data-and-code-licences) for more information on software licenses.

**Comment by cfchen98** [comment]
I would like to use CC BY-NC-SA lecense for my research.This license allows the distribution, remixing, adapting, and building upon the material in any medium or format while for noncommercial purposes only.

**Reply by EstherPlomp** [reply]
Sounds good! It also ensures that others will use the same license when they are building and sharing the original work (the SA part).

**Comment by arr0w-hs** [comment]
From the license chooser, i would use the CC BY-NC 4.0 license. It would allow reusers to mix, adapt and rebuild on my work, for non-commercial purposes only.

**Reply by EstherPlomp** [reply]
Sounds good! 

Additionally - you indicated that you also use code in your research: will you be able to share any of the code/scripts/software and if so - what is your preferred license? The licenses for software are a bit more complicated because software is more complex to reuse. See the [resources in the course materials](https://estherplomp.github.io/TNW-RDM-101/07-FAIR-principles.html#data-and-code-licences) for more information on software licenses.

**Comment by yzhang0429** [comment]
According to the license chooser I would choose CC BY-NC 4.0 license. With this license reusers have to give credit to the creator. And they can adapt or build upon the material for non commercial usage. If necessary, I may also use CC BY-NC-SA license to make sure the reusers use the same license as me.

**Reply by EstherPlomp** [reply]
Sounds good!

Additionally - you indicated that you also use code in your research: will you be able to share any of the code/scripts/software and if so - what is your preferred license? The licenses for software are a bit more complicated because software is more complex to reuse. See the [resources in the course materials](https://estherplomp.github.io/TNW-RDM-101/07-FAIR-principles.html#data-and-code-licences) for more information on software licenses.

**Comment by moengels** [comment]
I would choose CC BY-NC 4.0 for my imaging data, as it would anyway be necessary to give credit to the creator when describing sample preparation. Concerning code I would tend towards MIT as it is quite standard in the field and thus allows me to reuse code from others in my work and people can build on top of my code.

**Reply by EstherPlomp** [reply]
Hi @moengels! Thanks for sharing your license preferences! 

I think I forgot to update the link to the discussion session in the materials - so your thread is now situated in the closed discussion thread of the previous class. Can you please copy paste your comment to the open thread here: #34? Thanks and apologies about that!

Also a note on your choices: MIT is indeed a very permissive license. In terms of your data license choice I would have thought that you would prefer a copy left software license or that you would prefer the CC-BY licence instead of the CC BY-NC licence. The last one prohibits commercial reuse of the data. I hope this is indeed what you meant with choosing this license?
